vulnfeeds/cmd/alpine/fixtures/invalid_versions.txt

```diff
@@ -0,0 +1,7 @@
+1.18.-r2
```

Comment:
I've just skimmed https://github.com/alpinelinux/abuild/blob/master/APKBUILD.5.scd to try and better understand validity and invalidity.

From my skim, it's a bit light on details on how _pkgver_ and _pkgrel_ get glommed together, am I right?

It seems like _pkgrel_ itself is an ordinal number, and the "r" gets introduced as part of the concatenation with the _pkgver_ and the hyphen.

Based on that, any other letter than "r" would be, by this convention if not by the letter of what's documented, invalid. So maybe add a non-"r" version and also something without a hyphen (or underscore by the looks of it?) for completeness.

Oh wait, I see `2.19.10.4_p8` in the valid versions file.

Reply:
Added some more invalid versions examples.

`2.19.10.4_p8` specifically works because `p` is one of the allowed suffixes specified in the APKBUILD when paired with connected with an `_`. Another letter there would not work.